src/pages/Inventory.page.js

```diff
@@ -1,15 +1,80 @@
+import { getItemsListData } from '../additionalFunctions';
+
 const { BaseSwagLabPage } = require('./BaseSwagLab.page');
 
 export class InventoryPage extends BaseSwagLabPage {
     url = '/inventory.html';
 
-    get headerTitle() { return this.page.locator('.title'); } //
+    get addButton() { return this.page.locator('//button[contains(text(), "Add to cart")]'); }
```

Comment:
Its a good practice to use css over all project, not mixing css and xpath in one.

Reply:
this selector is unused, I'll delete it